src/ServerTelemetryChannel/SamplingTelemetryProcessor.cs

```diff
@@ -3,32 +3,34 @@
     using System;
     using System.Collections.Generic;
     using System.Threading;
-    
+
     using Microsoft.ApplicationInsights.Channel;
     using Microsoft.ApplicationInsights.DataContracts;
     using Microsoft.ApplicationInsights.Extensibility;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
     using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation;
-    
+
```

Comment:
need to update public comment for this public class, as this now handles pro-active sampling as well.

Reply:
As per Sergey's request, this experimental feature flag is currently only built for internal consumption. I can mention something like "This processor respects experimental feature flags related to sampling."

Reply:
Yea something like that would do. I'd prefer if the word pro-active sampling is mentioned as well. (its not really mentioned in official docs which SDK is used by code-less scenarios, so many people assume its the same sdk with same config.. ) May avoid some confusion.